src/modules/Projects/index.js

```diff
@@ -0,0 +1,129 @@
+import React from "react"
+
+import { FaGithub } from "react-icons/fa"
+import { FaMedium } from "react-icons/fa"
+import { FaTwitter } from "react-icons/fa"
+
+import {
+  SectionWrapperProjects,
+  ProjectBackground,
+  ProjectsWrapper,
+  ProjectListSection,
+  ProjectTopContainer,
+  ProjectBottomContainer,
+  ProjectCard,
+} from "./Layout"
+import {
+  ProjectCategoryListContainer,
+  ProjectCategoryList,
+  CategoryContainer,
+  CategoryChart,
+  CategoryNumber,
+  CategoryButton,
+  Category,
+} from "./Category"
+
+import {
+  ProjectTitle,
+  ProjectLabel,
+  ProjectLabelAudited,
+  ProjectDescription,
+  ProjectLinkContainer,
+  ProjectLink,
+  TitleLinkLearnMore,
+  IcoArrow,
+} from "./Basic"
+
+const Project = ({
+  hide,
+  children,
+  className,
+  learnMoreLink,
+  name,
+  github,
+  medium,
+  twitter,
+  description,
+  status,
+  country,
+  audited,
+  ...otherProps
+}) => (
+  <ProjectCard hide={hide} className={className}>
+    {children === null || children === undefined ? (
```

Comment:
I saw you make a reference to using `children` e.g.:

```The dot notation is entirely dependent on how you structure the index.js file inside the respective module. But most components have been exported so you can directly import them based on your own needs. (See Stat/index.js on how I've structured the component with consideration for it's children)```

Just for my own education, I assume you're referring to [this](https://reactjs.org/docs/react-api.html#reactchildren)?

Reply:
@as-dr Yep, so it's possible to just import ProjectCard with the default expectation of it's original layout or you can construct the contents of the component without having to make a new wrapper. 

For example, you can render a "Project" knowing that it will have a project card, and all of it's layout/children needs from the default as long as the right props are provided:

```
<Project {...project} />
```

or using your own child elements, but keeping the ProjectCard as your container:

```
<Project>
 <div>
  <p>Custom Children and Layout things</p>
 </diiv>
</Project>
```

These components are the "default" that gets pulled from the index.js file of a module, but it means you can create an almost "smart component" that's not dependent on it's children to render and can keep it's functionality consistent.